src/fontra/client/web-components/ui-form.js

```diff
@@ -171,6 +172,37 @@ export class Form extends SimpleElement {
     valueElement.appendChild(inputElement);
   }
 
+  _addEditAngle(valueElement, fieldItem) {
+    const inputElement = document.createElement("input");
```

Comment:
We have an "input" shortcut in html-utils.js. I'm pretty sure you can write this instead:
```js
    const inputElement = html.input({type: "number", "value": ...etc...});
```